package.json

```diff
@@ -43,6 +43,7 @@
     "chalk": "1.1.3",
     "cheerio": "0.22.0",
     "compression": "1.7.2",
+    "connect-session-knex": "1.4.0",
```

Comment:
This was quick and easy session persistence - I checked the code and it seems decent. but we might not wanna actually use this.